cdk/v2/utils.js

```diff
@@ -25,14 +24,6 @@ async function getWorkflowPath(
   };
 
   const workflowFilenames = flowTypeMap[flowType];
-  if (!workflowFilenames) {
-    throw new ErrorBuilder()
-      .setMessage(
-        "Unable to identify the workflow file. Invalid flow type input"
-      )
-      .setStatus(400)
-      .build();
-  }
```

Comment:
This is required as for an invalid flow type parameter, it'll error out on Ln 30. At least a different kind of validation is required.

Reply:
Addressed.